src/whereEq.js

```diff
@@ -0,0 +1,35 @@
+var _curry2 = require('./internal/_curry2');
+var eq = require('./eq');
+var mapObj = require('./mapObj');
+var where = require('./where');
+
+
+/**
+ * Takes a spec object and a test object; returns true if the test satisfies
+ * the spec, false otherwise. An object satisfies the spec if, for each of the
+ * spec's own properties, accessing that property of the object gives the same
+ * value (in `R.eq` terms) as accessing that property of the spec.
+ *
+ * `whereEq` is a specialization of [`where`](#where).
```

Comment:
`whereEq` is the generalisation of `where`, while `where` is a specialisation of `whereEq`.